contact.php

```diff
@@ -0,0 +1,108 @@
+<?php
+$name = $_POST['name'];
+$phone = $_POST['phone'];
+$email = $_POST['email'];
+$dropdown = $_POST['dropdown'];
+$message = $_POST['message'];
+ 
+$strValidationMessage = "";
+ 
+$to = "example@rawad.com";
+$subject = "New Message";
+ 
+if (isset($_POST['submit']))
+{
+  $name = $_POST['name'];
+  $phone = $_POST['phone'];
+  $email = $_POST['email'];
+  $message = $_POST['message'];
+ 
+  $boolValidateOK = 1;
+ 
+  if(trim($name)==="")
+  {
+    $boolValidateOK = 0;
+    $strValidationMessage .= "\nPlease Enter your name.<br />";
+  }
+ 
+ 
+    //Phone Number Valadation
+  $phone = str_replace("-","",$phone);
+  $phone = str_replace(".","",$phone);
+  $phone = str_replace("(","",$phone);
+  $phone = str_replace(")","",$phone);
+  $phone = str_replace(" ","",$phone);
```

Comment:
I would take the regex provided in your post to validate the phone numbers. You can use PHP's preg_match or preg_match_all to accomplish this.